rootfs/ingest/python/qserv/queue.py

```diff
@@ -63,13 +63,13 @@ def __init__(self, connection, chunk_meta):
         self.ordered_tables_to_load = self.chunk_meta.get_tables_names()
         _LOG.debug("Ordered tables to load: %s", self.ordered_tables_to_load)
         self.next_current_table()
-        chunk_files_count = self._count_chunk_files()
 
+    def set_transaction_size(self, chunk_queue_fraction):
+        """ Set number of chunk files managed by a single transaction
+        """
+        chunk_files_count = self._count_chunk_files()
         _LOG.debug("Chunk files queue size: %s", chunk_files_count)
-
-        # TODO add a parameter
-        nb_transaction = 10
-        tmp = int(chunk_files_count / nb_transaction)
+        tmp = int(chunk_files_count / chunk_queue_fraction)
```

Comment:
The name of the identifier `tmp` has no meaning. Could it be given some appropriate name to explain what's the content of the var plase?

Reply:
:+1: